api/src/services/import-services/csv-import-strategy.interface.ts

```diff
@@ -0,0 +1,83 @@
+import { WorkSheet } from 'xlsx';
+import { z } from 'zod';
+import { IXLSXCSVValidator } from '../../utils/xlsx-utils/worksheet-utils';
+
+/**
+ * Type wrapper for unknown CSV rows/records
+ *
+ */
+export type Row = Record<string, any>;
+
+/**
+ * Implementation for CSV Import Services.
+ *
+ * @description All CSV import services should implement this interface to be used with `CSVImportStrategy`
+ */
+export interface CSVImportService<ValidatedRow = Record<string, unknown>, InsertReturn = unknown> {
```

Comment:
nit: could add the @template jsdoc for the 2 generics

Reply:
fixed